engines/pytorch/pytorch-engine/build.gradle

```diff
@@ -14,12 +14,8 @@ dependencies {
 compileJava.dependsOn(processResources)
 
 processResources {
-    outputs.file file("${project.buildDir}/classes/java/main/pytorch-engine.properties")
-    doFirst {
-        def classesDir = file("${project.buildDir}/classes/java/main/")
-        classesDir.mkdirs()
-        def propFile = new File(classesDir, "pytorch-engine.properties");
-        propFile.text = "djl_version=${djl_version}\npytorch_version=${pytorch_version}"
+    filesMatching("**/pytorch-engine.properties") {
```

Comment:
was the `outputs.file` intentionally removed?

Reply:
In this case yes, it is part of the source tree now instead of being generated. (It is only the properties that are expanded).

There seemed to be 2 types of process resources tasks:
- this type, which was generating a version file. I simplified these to inject properties into a committed source file instead
-  another type which downloaded file(s) from URL(s) and processed them to be included in some way. Those I left as-is, just declaring inputs/outputs accordingly.